app/views/catalog/_index_default.html.erb

```diff
@@ -1,17 +1,27 @@
-<%# OVERRIDE Hyrax 3.4.2: change layout for metadata %>
+<%# OVERRIDE Hyrax 3.4.2: add conditional rendering and change layout for metadata %>
 
 <% doc_presenter = index_presenter(document) %>
-<%# default partial to display solr document fields in catalog index view -%>
-
-<div class="document-metadata">
-  <% collection_member = document['member_of_collections_ssim'] %>
-  <% doc_presenter = index_presenter(document) %>
-  <% if collection_member.present? %>
-    <p><%= render_collection_links(document) %></p>
-  <% end %>
-  <% index_fields(document).each do |field_name, field| %>
-    <% if field_name == 'member_of_collections_ssim' %>
-      <p><%= field %></p>
+<%# default partial to display solr document fields in catalog index view %>
+<% if search_results_theme == 'dc_repository_search' && home_page_theme == 'dc_repository' %>
+  <div class="document-metadata">
+    <% collection_member = document['member_of_collections_ssim'] %>
+    <% doc_presenter = index_presenter(document) %>
+    <% if collection_member.present? %>
+      <p><%= render_collection_links(document) %></p>
+    <% end %>
+    <% index_fields(document).each do |field_name, field| %>
+      <% if field_name == 'member_of_collections_ssim' %>
+        <p><%= field %></p>
+      <% end %>
+    <% end %>
+  </div>
+<% else %>
+  <dl class="document-metadata dl-horizontal dl-invert">
+    <% index_fields(document).each do |field_name, field| %>
+      <% if should_render_index_field? document, field %>
+        <dt class="blacklight-<%= field_name.parameterize %>"><%= render_index_field_label document, field: field_name %></dt>
+        <dd class="blacklight-<%= field_name.parameterize %>"><%= doc_presenter.field_value field %></dd>
+      <% end %>
     <% end %>
-  <% end %>
-</div>
+  </dl>
+<% end%>
```

Comment:
just a linting thing, missing a space after `end`

Reply:
Got it. Will fix now.